backend/routes/factions.go

```diff
@@ -202,3 +204,68 @@ func getFactions(w http.ResponseWriter, r *http.Request) {
 	}
 	routeutils.WriteDataJson(w, string(factions))
 }
+
+func getFactionMembers(w http.ResponseWriter, r *http.Request) {
+	factionID, err := strconv.Atoi(r.URL.Query().Get("factionID"))
+	if err != nil || factionID < 0 {
+		routeutils.WriteErrorJson(w, http.StatusBadRequest, "Invalid faction ID")
+		return
+	}
+
+	pageLength, err := strconv.Atoi(r.URL.Query().Get("pageLength"))
+	if err != nil || pageLength <= 0 {
+		pageLength = 10
+	}
+	if pageLength > 50 {
+		pageLength = 50
+	}
+
+	page, err := strconv.Atoi(r.URL.Query().Get("page"))
+	if err != nil || page <= 0 {
+		page = 1
+	}
+	offset := (page - 1) * pageLength
+
+	query := `
```

Comment:
For this query, can you add a JOIN statement with the `Users` table to get the user's username aswell as part of the return value.


Also, it is possible for multiple rows in `FactionMembersInfo` to have the same user address. Can you have it so the allocation is accumulated ( add up all allocations for all rows with the same user address )

So for example:
user address | allocation
0x12 | 3
0x24 | 4
0x12 | 2
0x35 | 1

Would give
user address | allocation
0x12 | 5
0x24 | 4
0x35 | 1